Sources/Client/Utils/InternetConnection.swift

```diff
@@ -50,7 +50,14 @@ public final class InternetConnection {
     private var lastState: State = .unknown
     private lazy var reachability: Reachability? = {
         do {
-            let reachability = try Reachability(hostname: Client.shared.baseURL.wsURL.host ?? "getstream.io")
```

Comment:
It should be
```swift
let reachability = try Reachability(hostname: Client.shared.baseURL.wsURL.host ?? "getstream.com")
```
per task description, not empty init

Reply:
We don't have getstream.com

Reply:
@tbarbugli I thought we should use `getstream.com`?